server/core/templates/login.html

```diff
@@ -15,24 +15,6 @@
 			required=true
 		/>
 
-		<input
```

Comment:
I thought this was here for password managers, idk if they're needed though

Reply:
Yes, the hidden inputs are for password managers.

Reply:
yeah I was gonna check with you first I figured they're there for a reason, but it kinda breaks the bootstrap's rounded corners. I need to get a little creative

Reply:
You might be able to shove them inside a div or somewhere else ? idk if they need to be next to eachother per-se

Reply:
Exactly what I'm considering 😄